bee-ledger/src/workers/consensus/worker.rs

```diff
@@ -240,38 +241,35 @@ where
     }
 
     async fn start(node: &mut N, config: Self::Config) -> Result<Self, Self::Error> {
-        let (snapshot_config, pruning_config) = config;
+        let (mut snapshot_config, mut pruning_config) = config;
 
         let (tx, rx) = mpsc::unbounded_channel();
 
         let tangle = node.resource::<MsTangle<N::Backend>>();
+        let tangle_config = node.resource::<TangleConfig>();
         let storage = node.storage();
         let bus = node.bus();
 
         validate_ledger_state(&*storage).await?;
 
-        // TODO should be done in config directly ?
```

Comment:
Maybe this comment confused you but I was going to do this another way as part of my ledger refactoring/cleaning. I prefer changing the conf values within the config `finish` than here with an override method.